docs/conf.py

```diff
@@ -55,6 +58,7 @@
 
 exclude_patterns = [
     "*/.ipynb_checkpoints/*",
+    "_build/**/*",
```

Comment:
To solve infinite loop?

Reply:
yup. still doesn't work perfectly but at least it helps